document_hosting/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+
+# Copyright 2018 Rémy Taymans <remytaymans@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Document Hosting',
+
+    'summary': """
+    Manage the documents of your cooperative.
```

Comment:
Suggestion: "Manage documents that can be published on website with `document_hosting_website`."